assembly/server/pom.xml

```diff
@@ -60,29 +60,9 @@
     </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-underfs-gcs</artifactId>
-      <version>${project.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.alluxio</groupId>
       <artifactId>alluxio-underfs-local</artifactId>
```

Comment:
should we do the same for local ufs?

Reply:
that will break integration tests.
also local UFS brings no extra dependencies, thus I don't see a reason to class load local UFS

Reply:
why does it break integration tests? can't they still depend on alluxio-underfs-local?

Reply:
if it isn't too difficult, i'd like to classload all UFSes just to be consistent